src/utils/isLikedByCurrentUser.js

```diff
@@ -0,0 +1,7 @@
+export const isLikedByCurrentUser = (post, user) => {
+  return post.likes.likedBy.find(
+    (currUser) => currUser.username === user.username
+  )
+    ? true
```

Comment:
Suggestion: double NOT (!!) can be used here instead of explicitly returning boolean value.

Reply:
Fixed this as well. Thanks for reviewing.